modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceCheckout.testcase

```diff
@@ -251,6 +251,95 @@ definition {
 		}
 	}
 
+	test AddEmptyOrderFromAUserAndCheckoutFromAnotherUserViaMiniCart {
+		property custom.properties = "jsonws.web.service.paths.excludes=${line.separator}company.security.strangers.verify=false";
+		property portal.acceptance = "false";
+
+		task ("Given a Minium site with a two buyer users") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+
+			CommerceEntry.addAccountEntryUser(
+				accountName = "Commerce Account",
+				accountType = "Business",
+				agreeToTermsAndAnswerReminderQuery = "true",
+				createNewAccount = "true",
+				requireReset = "false",
+				userEmailAddress = "buyer@liferay.com",
+				userFirstName = "Buyer",
+				userLastName = "User",
+				userRole = "Buyer",
+				userScreenName = "buyeruser",
+				userSiteMembership = "Minium");
+
+			CommerceEntry.addAccountEntryUser(
+				accountName = "Commerce Account",
+				agreeToTermsAndAnswerReminderQuery = "true",
+				createNewAccount = "false",
+				requireReset = "false",
+				userEmailAddress = "buyer2@liferay.com",
+				userFirstName = "Buyer",
+				userLastName = "User 2",
+				userRole = "Buyer",
+				userScreenName = "buyeruser2",
+				userSiteMembership = "Minium");
+		}
+
+		task ("Given buyer user 1 logs in") {
+			User.logoutPG();
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "buyer@liferay.com");
+		}
+
+		task ("And given an empty order created from Pending Orders") {
+			CommerceNavigator.gotoMiniumSidebarItem(menuItem = "Pending Orders");
+
+			Click.mouseDownMouseUp(
+				key_text = "Add Order",
+				locator1 = "Button#ANY");
+		}
+
+		task ("When buyer user 2 logs in navigating in Minium site") {
+			User.logoutPG();
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "buyer2@liferay.com");
+		}
+
+		task ("And buyer user 2 adds a product to the cart for the order opened by buyer user 1") {
+			FrontStore.addInitializerProductToCart(
+				productName = "U-Joint",
+				productQuantity = "1");
+		}
+
+		task ("And by clicking Submit button from minicart, comletes the checkout process") {
+			CommerceNavigator.gotoMiniCart();
+
+       		Button.click(button = "Submit");
```

Comment:
This will probably not work on CI because of the known resolution issue that affects the Minium sidebar on CI.
Please consider adding a separate macro for these lines.
Add a `ScrollBy(value1 = "0, 1000");` that will scroll the page down and move the sidebar that covers the Submit button. Thanks.